nixos/modules/config/users-groups.nix

```diff
@@ -648,7 +648,7 @@ in {
         install -m 0700 -d /root
         install -m 0755 -d /home
 
-        ${pkgs.perl.withPackages (p: [ p.FileSlurp p.JSON p.DateTime ])}/bin/perl \
+        ${pkgs.perl.withPackages (p: [ p.FileSlurp p.JSON p.TimePiece ])}/bin/perl \
```

Comment:
I don't even need a module for that, it's in core: https://perldoc.perl.org/Time::Piece

```
use Time::Piece;

my $t = localtime;
print "Time is $t\n";
print "Year is ", $t->year, "\n";
```

Reply:
```suggestion
        ${pkgs.perl.withPackages (p: [ p.FileSlurp p.JSON ])}/bin/perl \
```